stablehlo/tests/vhlo/stablehlo_legalize_to_vhlo.0_10_0.mlir

```diff
@@ -5,6 +5,10 @@
 // RUN: diff %t.0 %t.1
 // RUN: stablehlo-translate --serialize --target=0.10.0 --strip-debuginfo %s > %t.2
 // RUN: diff %s.bc %t.2
+// RUN: stablehlo-opt --stablehlo-legalize-to-vhlo -emit-bytecode -debug-only=vhlo-bytecode %s 2>&1 | FileCheck --check-prefix=CHECK-WARN %s
```

Comment:
Also can you update `vhlo.md` with the new test header? 

https://github.com/openxla/stablehlo/blob/main/docs/vhlo.md#add-versioned-serialization-test

Reply:
Done.

Reply:
Actually, reading these instructions again I remember what happened now. I followed the instructions in the doc you linked, but I forgot to do the manual part at the end. IMO we should advise to copy the highest versioned test and add tests to it, I'll update the doc.